hdsemg_pipe/widgets/WizardStepWidget.py

```
@@ -0,0 +1,200 @@
+        raise NotImplementedError("Subclasses must implement the check method.")
+
+    def complete_step(self):
+        """Marks the step as completed."""
+        self.success(f"Step {self.step_index} completed successfully!")
+        self.step_completed = True
+        global_state.complete_widget(f"step{self.step_index}")
+        self.stepCompleted.emit(self.step_index)
+
+    def setActionButtonsEnabled(self, enabled, override=False):
+        """Enables or disables action buttons."""
+        if enabled == True and global_state.is_widget_completed(f"step{self.step_index - 1}") or enabled == False or override:
```

Ruff (0.14.6)

Avoid equality comparisons to `True`; use `enabled:` for truth checks

Replace with `enabled`

(E712)